templates/nrpe.cfg.erb

```diff
@@ -149,6 +149,7 @@ command[check_postgres]=/usr/lib64/nagios/plugins/check_postgres $ARG1$
 command[check_raid]=/usr/lib64/nagios/plugins/check_raid $ARG1$
 command[check_time_skew]=/usr/lib64/nagios/plugins/check_time_skew $ARG1$
 command[check_cpu]=<%= @nrpe_local_script_path %>/check_cpu $ARG1$
+command[check_http]=/usr/lib64/nagios/plugins/check_http $ARG1$
```

Comment:
We usually do not add the checks here, right?

Reply:
This is for the "behind haproxy/localhost" testing case, the idea would be to use the generic http check installed by the module rather than reinvent the wheel. I'm not entirely sure where else to add a thing like this. Custom checks would go into a dedicated .cfg file for sure

Reply:
Ok